src/api/index.ts

```diff
@@ -1,7 +1,14 @@
 import axios from 'axios';
 
+
+
 const api = axios.create({
-	baseURL: 'http://localhost:5001/'
+	baseURL: 'http://localhost:5001/',
 });
 
+const repoAdoptToken = localStorage.getItem('repoAdoptToken') ?? false;
+if(repoAdoptToken){
+	api.defaults.headers.common['Authorization'] = repoAdoptToken;
+}
+
```

Comment:
I would suggest to use an interceptor
Right now this works since we use API after we are logged in and have an item in the local storage, however if API is used before this, it will not update the variable